codegen_glibmm/templates/proxy.cpp.templ

```diff
@@ -127,7 +127,9 @@ void {{ class_name_with_namespace }}::{{ prop.name }}_set({{ prop.cpptype_in }}
     m_proxy->call("org.freedesktop.DBus.Properties.Set", cb, params);
 }
 
-void {{ class_name_with_namespace }}::{{ prop.name }}_set_finish(const Glib::RefPtr<Gio::AsyncResult>&/* res */) {
+void {{ class_name_with_namespace }}::{{ prop.name }}_set_finish(const Glib::RefPtr<Gio::AsyncResult>& result) {
+    Glib::VariantContainerBase wrapped;
+    wrapped = m_proxy->call_finish(result);
```

Comment:
could you remove the unnecessary local variable please.
`m_proxy->call_finish(result);`

and of course adopt the test accordingly